server/config.ts

```diff
@@ -114,6 +114,19 @@ export default {
       },
       agent: new AgentConfig(Number(get('BREACH_NOTICE_API_TIMEOUT_RESPONSE', 5000))),
     },
+    ndeliusDeeplink: {
+      url: get(
+        'NDELIUS_DEEPLINK_URL',
+        'http://localhost:7001/NDelius-war/delius/JSP/deeplink.xhtml',
+        requiredInProduction,
+      ),
+      healthPath: '/health/ping',
+      timeout: {
+        response: Number(get('NDELIUS_INTEGRATION_TIMEOUT_RESPONSE', 10000)),
+        deadline: Number(get('NDELIUS_INTEGRATION_TIMEOUT_DEADLINE', 10000)),
+      },
+      agent: new AgentConfig(Number(get('NDELIUS_INTEGRATION_TIMEOUT_RESPONSE', 10000))),
```

Comment:
You don't need these lines - they're just for API clients. Might be worth moving the `ndeliusDeeplink` bit out of the `apis` block actually?
```suggestion
```